04_numeronym/jessodri/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"unicode"
+)
+
+var out io.Writer = os.Stdout
+
+func isASCII(s string) bool {
```

Comment:
It probably simpler to support unicode than test for multibyte.

Reply:
Probably, but I'm happy to leave it how it is for now.

Reply:
I agree, I think this is complicating things more than what was intended. I don't believe it asked for only strings involving ascii code.

> Write a function that returns a slice of numeronyms for its input strings